tsconfig.test.base.json

```diff
@@ -2,7 +2,6 @@
   "extends": "./tsconfig.base",
   "compilerOptions": {
     "noEmit": true,
-    "lib": ["es2017", "esnext.asynciterable"],
```

Comment:
Btw, I think it's fine to just add `es2019.array` here, if you were trying to avoid doing that.

Reply:
I was, since I didn't really want to mix the polyfill impl. details of the `apollo-federation` package with the rest of the monorepo's. Is this a non-issue (at least for testing)?